src/annotator-content-analysis.js

```diff
@@ -80,6 +80,19 @@ function annotatorContentAnalysis(options){
 
         })
 
+        $('.commit-question').on('click', (e) => {
```

Comment:
no, i think we want UI interaction elements in `form.handlebars` and apply a class to the question in focus.